### ISSUE TITLE
Adapt Hidden Hands processing scripts to their v2

### ISSUE THREAD

**Issue by sajanrajdev** [Badger-Finance/badger-multisig#1335]
Hidden Hands just released a [V2](https://mirror.xyz/0xE90c74145245B498fef924fAdC7bb34253c7cF90/uMF43OIe5k3hF-D6ZeLUWkoD-iOAv-HNYzmhgT8kkcg) of their contracts. The flows and scripts for claiming and and processing graviAURA's bribes must be fixed. These are the new addresses: 
```
Mainnet

RewardDistributor | 0xa9b08B4CeEC1EF29EdEC7F9C94583270337D6416
RewardSwapper | 0x4177b6f4C38A2ee94606e10049691393643b80ae
RewardHarvester | 0xd23aa7EdF42CD3Fc4CD391faAbc0c207B1c86542
RewardHarvesterClaim | 0x4666f33E4fbE6C8AA9671bD2AEdc46d641451F85
BribeVault | 0xE00fe722e5bE7ad45b1A16066E431E47Df476CeC
BribeMarket (implementation) | 0x0F7983eb3D146477B6c7d4C5aAB427D3340DB38E
BribeFactory | 0xB49f6D281b84D3B29203BA38360a7cb97780B2f1

BribeMarket for BALANCER | 0x45Bc37b18E73A42A4a826357a8348cDC042cCBBc
BribeMarket for AURA | 0xcBf242F20D183B4116C22Dd5e441b9aE15b0d35A
BribeMarket for FRAX | 0x2694214a4EA6A2C9A9427AA12251705B67e86960
BribeMarket for BUNNI | 0x175b01eF7CEa8615B1D9485BC827378EA5D50C86
BribeMarket for FLOOR_DAO | 0x19302c4ec309b3913e8bE41025DCF66A562BDa42
BribeMarket for RIBBON | 0x97A45f52e5EdD775923a8DE3f5373D41DA5dDa73
BribeMarket for PENDLE | 0xf12e79c95d3C13e2ab4f8222d698DAA53Df17aD8
```

**Comment by sajanrajdev** [comment]
# Impact Analysis on Bribes Processing

## GraviAURA Strategy
- Interaction with HH occurs in the following function: https://github.com/Badger-Finance/vested-aura/blob/f05bfb22cb82ecaa57cd957ce7fe530e900e8384/contracts/MyStrategy.sol#L386
- Function takes the rewards distributor as a parameter (Great because this address changed)
- Diff of RewardsDistributor.sol: https://www.diffchecker.com/Jvc22SeX/

NOTES:
- Main difference on rewards distributor is style and the introduction of pausability (Owned by [RedactedCartel.eth](https://etherscan.io/address/0xA52Fd396891E7A74b641a2Cb1A6999Fcf56B077e) - governance multi, 3/6)
- `rewards` mapping is only used to fetch the "token" address for each reward - untouched
- Claiming function remains virtually untouched
BREAKING CHANGE: `BRIBE_VAULT` constant removed in the new version. An upgrade is required to remove this query, otherwise the function will revert.

**Comment by gosuto-inzasheru** [comment]
does this mean redeployment of the strategy or of the processor?

**Comment by sajanrajdev** [comment]
> does this mean redeployment of the strategy or of the processor?

It would mean of the strategy. The processor is blind to HH. 

I am already working on the fixes to the strategy but since a Timelock is required, it won't be ready for this week's processing. For this reason, I propose that we subsidize this round's bribes (only voting for `MetaStable wstETH/WETH` so we can expect around ~4.5K USDC obeing received as the only bribe amount) and recover it after the upgrade.

**Comment by sajanrajdev** [comment]
> > * BREAKING CHANGE: CHANGE: BRIBE_VAULT constant removed in the new version. An upgrade is required to remove this query, otherwise the function will revert.
> 
> This var was used for signalling vanilla eth token/transfers, meaning that probably now it is not supported anymore. cc: @sajanrajdev

Indeed, native tokens are no longer supported and, therefore, this signaling is no longer applied. Confirmed with the HH team.

**Comment by petrovska-petro** [comment]
@sajanrajdev , agree with the findings described. upgrade is minor on lack of `BRIBE_VAULT` in v2. structs args and *claim* method at least facing abi is same as prev
